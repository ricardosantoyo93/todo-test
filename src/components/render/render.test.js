import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ShallowRenderer from 'react-test-renderer/shallow';

import { Render } from './index';

Enzyme.configure({ adapter: new Adapter() });

describe('Render', () => {
  let wrapper = null;

  beforeEach(() => {
      wrapper = shallow(<Render />);
  });
  
  it('matches snapshot', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<Render />);
    expect(tree).toMatchSnapshot();
  });

  test('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });

  test('should be titled Done', () => {
    wrapper = shallow(<Render source="done" done={[]} />);
    expect(wrapper.contains(<h3>Done</h3>)).toBe(true);
  });
});
