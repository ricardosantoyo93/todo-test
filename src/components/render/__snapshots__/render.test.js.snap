// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render matches snapshot 1`] = `
<React.Fragment>
  <span
    className="cards-content"
  >
    <h3>
      To Do
    </h3>
    <UNDEFINED />
  </span>
</React.Fragment>
`;
